code/defines/procs/admin.dm

```diff
@@ -39,14 +40,20 @@
 
 	. = ""
 
+	if(check_watchlist(key) && is_admin(src))
```

Comment:
```
code/defines/procs/admin.dm, line 43, column 29:
error: undefined var: "src"
```
`src` is null here because this proc isnt attached to a mob or client, its just a bare proc with no parent.